### ISSUE TITLE
Updating an existing order produces an error.

### ISSUE THREAD

**Issue by JustinSainton** [zao-web/zao-woocommerce-quickbooks-online-integration#6]
```
[22-Sep-2017 20:10:15 UTC] PHP Warning:  call_user_func_array() expects parameter 1 to be a valid callback, class 'QuickBooksOnline\API\DataService\DataService' does not have a method 'create_product' in /dom10096/wp-content/plugins/zao-woocommerce-quickbooks-online-integration/vendor/zao-web/qbo-connect/lib/service.php on line 177
```

**Comment by jtsternberg** [comment]
Where do you see this? Updating existing orders is working fine for me.

**Comment by JustinSainton** [comment]
Fixed in 632425c54f0a3ea5c9684902819e2d6d85de5ae5
